boltrouter/bolt_request.go

```diff
@@ -215,21 +259,33 @@ func (br *BoltRouter) DoRequest(logger *zap.Logger, boltReq *BoltRequest) (*http
 		return nil, false, boltRequestAnalytics, err
 	}
 
-	logger.Debug("initial request target", zap.String("target", initialRequestTarget), zap.String("reason", reason))
+	logger.Debug("initial request target", zap.String("target", InitialRequestTargetMap[initialRequestTarget]), zap.String("reason", reason))
 
-	if initialRequestTarget == "bolt" {
+	if initialRequestTarget == InitialRequestTargetBolt {
 		resp, isFailoverRequest, err := br.doBoltRequest(logger, boltReq, false, boltRequestAnalytics)
 		// if nothing during br.doBoltRequest panics, err will not be of type ErrPanicDuringBoltRequest so failover was
 		// handled inside the function as needed, and we can just return
 		// If the err is of type ErrPanicDuringBoltRequest then we need to failover to AWS manually since .doBoltRequest
 		// halted execution before it could failover
 		if err != nil && errors.Is(err, ErrPanicDuringBoltRequest) && br.config.Failover {
```

Comment:
Why do we have a `ErrPanicDuringBoltRequest` ? we should return errors not wrap panics, I am confused when reading the code why this was added, seems anti-go

Reply:
`ErrPanicDuringBoltRequest ` was added in https://github.com/project-n-oss/sidekick/pull/70 with the goal of always returning a sensible output to the client.

The idea behind catching panics is that if a replica goes away mid-request, the `http.Do` panics with connection broken vs returning an error so that we can failover or return a sensible (most likely 500) error to the client.